client/utils/auth.py

```diff
@@ -19,12 +19,26 @@
 
 log = logging.getLogger(__name__)
 
+
 CLIENT_ID = \
     '931757735585-vb3p8g53a442iktc4nkv5q8cbjrtuonv.apps.googleusercontent.com'
 # The client secret in an installed application isn't a secret.
 # See: https://developers.google.com/accounts/docs/OAuth2InstalledApp
 CLIENT_SECRET = 'zGY9okExIBnompFTWcBmOZo4'
-REFRESH_FILE = '.ok_refresh'
+
+
+def get_config_directory():
+    if sys.platform == 'win32':
```

Comment:
`~/.config/ok` probably makes more sense, especially since Windows users will generally be using a Unix-style shell like Git Bash anyway.